openmetadata-service/src/main/java/org/openmetadata/service/resources/search/SearchResource.java

```diff
@@ -173,6 +173,91 @@ public Response search(
     return searchRepository.search(request);
   }
 
+  @GET
```

Comment:
since this code is exactly the same as the other search api endpoint, we can do this via some queryParam only , and deviate the call if this is a hierarchy call or now search

Reply:
added a new query param in existing search api , as per feedback